notebooks/Hugging_Face_Finetuning.ipynb

```diff
@@ -0,0 +1,920 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # make to choose a tokenizer and bert model that work together
</pre><p>Is the following better wording?</p><p> </p><blockquote>We choose a tokenizer and bert model that works together</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/skorch-dev/skorch/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>